src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```diff
@@ -62,13 +62,13 @@
       Text="Cross-OS native compilation is not supported. https://github.com/dotnet/corert/issues/5458" />
 
     <Error Condition="'$(DisableUnsupportedError)' != 'true' and !($(RuntimeIdentifier.EndsWith('x64')) or
```

Comment:
We have `DisableUnsupportedError` to allow suppressing the error message. Adding the new platform to the list here is one of the last steps once everything is working.

Reply:
Make sense.

Reply:
General question. Does `DisableUnsupportedError` is `dotnet/runtime` property, or purely NativeAOT construct?

Reply:
It is NativeAOT specific.